packages/components/src/context-tag/context-tag.tsx

```diff
@@ -33,16 +36,28 @@ type Props = {
   | { type: 'address'; address: string }
   | { type: 'icon'; icon: React.ReactElement; label: string }
   | { type: 'audio'; audioLength: string }
+  | { type: 'label'; children: string }
```

Comment:
Await https://github.com/status-im/status-website/pull/213/files#r1311265478

Reply:
Another type "Label" added to https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Desktop%2FWeb?type=design&node-id=18409-18719&mode=design&t=gYqNYTsPL7i7Np3U-4.